### DISCUSSION TITLE
Exporting fits

### DISCUSSION THREAD

**Discussion by tjnhooper** [smeerten/ssnake#131] [Q&A]
Hey!

Thanks for the software, its great to have a usable program to generate disordered quadrupolar fitting. New to the software so hopefully my question isn't too inane. 

I cannot seem to export spectral fits as anything other than .png/.jpeg. I see the option to export to .svg/.eps/.pdf but no file is created. I've tried lowering the DPI just in case it was too large but that didn't help. 

Alternatively, is there anyway to export the simulated curve to a .csv or similar? I see how you do it for a processed spectrum but not for a fitted one. 

I'm using ssNake v1.4 via the windows 64bit installer on Windows 10. 

Thanks again!
Tom

**Comment by wfranssen** [comment]
Hi Tom,

**I cannot seem to export spectral fits as anything other than .png/.jpeg. I see the option to export to .svg/.eps/.pdf but no file is created. I've tried lowering the DPI just in case it was too large but that didn't help.**
I now verified this as a bug in the windows compile/installer I made. Something went wrong with bundling the right Python libraries. I found the fix for this. We will probably make a new installer (say version 1.4.1) that fixes this. Stay tuned for that. Thanks for reporting this issue!

**Alternatively, is there anyway to export the simulated curve to a .csv or similar? I see how you do it for a processed spectrum but not for a fitted one.**
In the fitting interface, there is a button 'Import/Export'. From there choose 'Curves to Workspace'. In this way you get a new workspace with the original and fitted data. From this workspace you can export the data to .csv.

Hope this helps.

Regards,

Wouter

**Reply by tjnhooper** [reply]
Hey Wouter, 

-Thanks for the quick response, I'll wait for the new release.
-Ah, I tried that option but missed the button to toggle between original/fitted/difference data slices. That works as well. 

Cheers,
Tom

**Comment by IgoiuS** [comment]
Hi,

I have the same problem using the "windows installer" version. I cannot export a stack plot (for instance) as .svg or .pdf to further work on it using Illustrator. However, if you download and compile the source code everything works. 

Best,

**Comment by wfranssen** [comment]
Hi IgoiuS,

Yes, the issue is still there in the installer. It will only be fixed when we release a new version (hopefully soon).

Regards,

Wouter
